### DISCUSSION TITLE
Katil kim sorusu?

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#214] [Zeka Sorusu] [answered]
Ali, Buse ve Kerem cinayet zanlısıdır.
Yapılan araştırmalarda cinayet esnasında katilin alkollü olduğu anlaşılmıştır. İçlerinden birinin katil olduğu bu üç zanlı sorgulamaları sırasında aşağıdaki ifadeleri vermişlerdir:
Ali: O gece alkollü değildim, ben katil değilim.
Buse: O gece alkollüydüm ama katil değilim.
Kerem: O gece alkollü değildim, Katil alkol kullanan birisidir.
Polisler daha sonra şu sonuca ulaşırlar:

Yukarıdaki ifadelerden sadece ikisi doğrudur.
Zanlılardan ikisi o gece alkollüdür.
Katil kim?

**Comment by AcelyaP** [comment, marked as answer]
Buse'yi eliyoruz çünkü iki kişi alkollü olması lazım ve bu yüzden doğru söyleyenler ali ve kerem olamaz çünkü ikisi de alkollü olmadığını savunuyor.
Kerem katilin alkol kullandığını söylüyor ve bu bilgi teyitlenmiş o yüzden Kerem ve Buse doğru söylüyor.
 **Ali katildir.**
